_includes/head/custom.html

```diff
@@ -3,3 +3,22 @@
 <link rel="icon" type="image/png" href={{ "/assets/images/icon-32.png" | relative_url }} sizes="32x32">
 <link rel="icon" type="image/png" href={{ "/assets/images/icon-96.png" | relative_url }} sizes="96x96">
 <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&amp;display=swap&amp;text=ROOTDataAnlysisFramework" rel="stylesheet">
+<head>
+  <link rel="manifest" href={{ "/manifest.json" | relative_url }}/>
```

Comment:
That gives `<link rel="manifest" href=/626/manifest.json/>` so we need
```suggestion
  <link rel="manifest" href="{{ "/manifest.json" | relative_url }}" />
```